test-dxcompiler-in-project/Makefile

```diff
@@ -1,12 +1,18 @@
 # TODO: find a way to download from draft release URLs; the below works for final releases.
 # For draft releases, save jar manually to resources/home/dnanexus/dxCompiler.jar
-DXCOMPILER_JAR_URL=https://github.com/dnanexus/dxCompiler/releases/download/2.7.0/dxCompiler-2.7.0.jar
+VERSION=2.8.3
+DXCOMPILER_PRERELEASE_PATH=project-G7GFgjQ0X386z5ZF24vvyFzp:dxCompiler-${VERSION}.jar
```

Comment:
Is there code somewhere that uploads the prerelease jar to this project? I don't see it. @commandlinegirl has also suggested that uploading the jar as a GitHub artefact (similar to what we do with snapshots) could be a workaround to be able to upload / download drafts. Any reason to prefer putting it on the platform vs. on GitHub? @jdidion  @commandlinegirl

Reply:
Nope, it's a manual process. I created a project on prod to upload pre-release versions for internal testing prior to release. I don't think we want to be making pre-release versions available publicly.